test/integration/volume/attach_detach_test.go

```diff
@@ -168,6 +170,18 @@ func waitToObservePods(t *testing.T, podInformer cache.SharedIndexInformer, podN
 	}
 }
 
+func waitForPodsInDSWP(t *testing.T, dswp volumecache.DesiredStateOfWorld) {
+	if err := wait.Poll(10*time.Second, 30*time.Second, func() (bool, error) {
```

Comment:
I will change the polling interval to 500ms.. Since `GetPodToAdd` requires lock and stuff - I don't want to cause any lock contention and 500ms should be good enough.